README.md

```diff
@@ -4,110 +4,167 @@
 
 # Disclaimer
 
-This project is currently on an experimental stage. We periodically synchronize the source code and test cases of this
-repository with the [CLI folder](https://github.com/apache/incubator-openwhisk/tree/master/tools/cli/go-whisk-cli) and the [test folder](https://github.com/apache/incubator-openwhisk/tree/master/tests) in OpenWhisk. The framework of test cases is under construction
-for this repository. Please contribute to the [CLI folder](https://github.com/apache/incubator-openwhisk/tree/master/tools/cli/go-whisk-cli) in OpenWhisk for any CLI changes, before we officially announce the separation
+This project is currently on an experimental stage. We periodically synchronize
+the source code and test cases of this repository with the [CLI
+folder](https://github.com/apache/incubator-openwhisk/tree/master/tools/cli/go-whisk-cli)
+and the [test
+folder](https://github.com/apache/incubator-openwhisk/tree/master/tests) in
+OpenWhisk. The framework of test cases is under construction for this
+repository. Please contribute to the [CLI
+folder](https://github.com/apache/incubator-openwhisk/tree/master/tools/cli/go-whisk-cli)
+in OpenWhisk for any CLI changes, before we officially announce the separation
 of OpenWhisk CLI from OpenWhisk.
 
+The OpenWhisk Command Line Interface (OpenWhisk CLI) is a unified tool that
+provides a consistent interface to interact with OpenWhisk services. With this
+tool to download and configure, you are able to manage OpenWhisk services from
+the command line and automate them through scripts.
 
-The OpenWhisk Command Line Interface (OpenWhisk CLI) is a unified tool that provides a consistent interface to
-interact with OpenWhisk services. With this tool to download and configure, you are able to manage OpenWhisk services
-from the command line and automate them through scripts.
+# Where to download the binary of OpenWhisk CLI
 
+The OpenWhisk CLI is available on the release page: [click here to
+download](https://github.com/apache/incubator-openwhisk-cli/releases). We
+currently have binaries available for Linux, Mac OS and windows under i386 and
```

Comment:
`Windows` instead of `windows`.

Reply:
Also a few sentences in here that start with double spaces. I know, I know.

Reply:
Fixed.